### ISSUE TITLE
MS4: Docked side panel tabs don't update to reflect which panel is shown

### ISSUE THREAD

**Issue by lonetech** [musescore/MuseScore#13945]
OS: Debian GNU/Linux bookworm/sid, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-3305804646, revision: github-musescore-musescore-8397f17

Method to reproduce:
Menu: View -> Selection Filter
Observe that selection filter panel has opened, but the tab above still indicates e.g. Palettes
Similarly, closing a tab (... -> Close) displays one of the other panels, but does not select the corresponding tab. 

Very minor bug. 

Edit: After I assigned a hotkey (F6) to Selection Filters, and used that to close the tab, the tabs have started updating. So there are conditions when it does work, but I am not certain when.

**Comment by Tantacrul** [comment]
Can you capture a video or take a screenshot? I'm having trouble understanding the precise nature of the visual bug. Thanks!

**Comment by lonetech** [comment]
Annoyingly, I can't. As noted in the edit, it stopped doing this at some point. I've tried deleting the setting files, starting the program over, docking and undocking panels, but the issue just isn't showing itself again, and I don't know why. Sorry about the unreproducable heisenbug report :(

The visual appearance is simple: from the default layout, on the left side, we have docked panels Palettes, Properties, and Instruments. I added Filters by View - Selection Filter. The Filter panel appeared, but the tab above for Palettes remained visibly selected, not the one for Filters which had just appeared. Similarly, when closing the Filters panel, another panel would appear but no tab showed as selected. 

I have tried rm -r ~/.config/MuseScore/MuseScore4Development.ini ~/Documents/MuseScore4Development ~/.local/share/MuseScore/MuseScore4Development/ for a clean slate but this didn't make the bug reappear. It could be related to something else unclear, like screen placements, wayland wm/compositor (Sway), keyboard layout... I really have no idea.
